### PR TITLE
libSplash: Add 1.6.0 Release

### PR DESCRIPTION
**Pull request by ax3l**
Add the latest release of libSplash, version 1.6.0.

### CODE REVIEW

**Comment by tgamblin** [comment]
flake8

**Comment by adamjstewart** [comment]
@tgamblin That looks like a regression in the url line length exceptions. @ax3l you may have to rebase to get the updated `spack flake8` that fixes this (I assume that is what is going on). Otherwise, @tgamblin should be able to merge this just fine, as it isn't actually a flake8 problem.

**Comment by tgamblin** [comment]
@ax3l: do you mind rebasing just to be sure?

**Comment by adamjstewart** [comment]
Hopefully we didn't forget to put an exception for lines that start with `git=`

**Comment by tgamblin** [comment]
@adamjstewart: I used your original regexes.  Looks like we have `version()`, `url=`, but not `git=`, so it'll fail when on a separate line like this.

**Comment by adamjstewart** [comment]
@tgamblin I thought it did lines containing `(http|https)` too?

**Comment by tgamblin** [comment]
@adamjstewart: true!

**Comment by tgamblin** [comment]
rebased.

**Comment by adamjstewart** [comment]
Hmm, definitely a bug in `spack flake8`. I tested it locally on the latest develop and it isn't working.

**Comment by ax3l** [comment]
Currently away from keyboard, thx for rebasing!

File: lib/spack/spack/cmd/flake8.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
     # exemptions applied to all files.
     r'.py$': {
         # Exempt lines with URLs from overlong line errors.
-        501: [r'^(https?|file)\:']
+        501: [r'(https?|file)\:']
```
Comment (tgamblin) [inline lib/spack/spack/cmd/flake8.py]: Just added them for the `package.py` files

**Comment by ax3l** [comment]
wuhu, it passes flake8 :)

**Pull request merged**